### ISSUE TITLE
About the accuracy of Voxelizer

### ISSUE THREAD

**Issue by RIA1159** [Ruyi-Zha/r2_gaussian#59]
Greetings!   I really appreciate your great work, it solve my problem to transfer gaussian into volume. 
And I have some problem with the voxelizer. I found that after training, the rendered images look quite precise and accurate, but when I voxelize the gaussian points to volume, the structure of the volume is very rough and imprecise. There seems to be a loss of detail in the voxelization process.

This is the rendered image from training, the details seem quite precise.

<img width="598" height="594" alt="Image" src="https://github.com/user-attachments/assets/9762944e-6555-43d8-9cc0-386d989b43c5" />

And this is the projection of the volume, it is very blurry, and the volume also has no structure details.

<img width="599" height="573" alt="Image" src="https://github.com/user-attachments/assets/12535f57-2bf5-477b-bbb0-928ddf1a8ae6" />

And here is my settings to get the volume:

D = cfg.img_size
    tv_vol_center = torch.tensor([0., 0., 0.])
    tv_vol_nVoxel = torch.tensor([D, D, D])
    tv_vol_sVoxel = torch.tensor([1., 1., 1.])

    vol_pred = query(
        gaussians,
        tv_vol_center,
        tv_vol_nVoxel,
        tv_vol_sVoxel,
        pipe,
    )["vol"] 
    vol_pred = vol_pred.cpu().numpy()   

Is there any wrong with my settings ？ Or how can I solve this problem?
Looking forward for your reply. :)

**Comment by Ruyi-Zha** [comment]
Hi, thanks for your interest in our work. I was wondering how the projections of the volume are generated. I guess you use a toolbox like TIGRE? A possible reason for blurry projection is that the volume has relatively low resolution. You might try increasing `tv_vol_nVoxel` to see if that improves the projection quality.

**Comment by RIA1159** [comment]
Thanks for your suggestion! 
I try to enlarge the tv_vol_nVoxel and get a larger volume. However, the projection still looks blurry, and the volume  has no structure details.
The projection and the volume: 

<img width="728" height="690" alt="Image" src="https://github.com/user-attachments/assets/9642ca80-b69e-4efb-ace1-399ca195a258" />

<img width="887" height="951" alt="Image" src="https://github.com/user-attachments/assets/71f8eac3-ef18-45d9-b0ff-a24006959e17" />

I visualize the volume with  ChimeraX. And I obtain the projection by accumulating the density values in the z-axis direction of the volume with the following code:

projection = np.mean(vol, axis = 2)

I wonder is this accumulation operation consistent with the logic in x-ray rasterization.

**Comment by RIA1159** [comment]
Thanks ! I solve the problem by setting the scaling_modifier to 0.5. 
Thanks for your time and patience! :)
